roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -43,6 +43,10 @@
     creates: "{{ texLive_dest }}/tlpkg/texlive.profile"
   tags: ngi_reports
 
+- name: Ugly TexLive permissions fix 
+  file: path="{{ texLive_dest }}" state="directory" mode="g+rwxs" recurse="yes" 
```

Comment:
Same thing here. Add a capital `X`. 

Sadly this will not work though, that's why I suggested you added the `find` snippet that I sent you, because your `file` task will be executed both when e.g. you and I run this. And because you have now installed these files on irma3 this task would completely fail for me, because I don't have permission to change this. I.e. you can only change permissions for files you own yourself, which the previously mentioned snippet solves.